metricbeat/module/golang/expvar/_meta/docs.asciidoc

```diff
@@ -0,0 +1,5 @@
+=== golang expvar MetricSet
+
+This is the expvar metricset of the module golang.
+Golang can exposes it's variables by expvar api, with this metricset, user can collect these metrics,
+collect all the expvar exposed variables except memstats(if you need memstats, check out another `heap` metricset in this module.
```

Comment:
Can you link here to the expvar docs from Golang perhaps? So in case someone has a golang app and wants to expose expvars, he gets some intructions on how to do it.